Cargo.toml

```diff
@@ -16,19 +16,23 @@ actix-http = "2"
 actix-rt = "1"  # 2+ breaks testing, May need actix-web 4+?
 actix-web = "3"
 backtrace = "0.3"
-cadence = "0.24"
+cadence = "0.25"
+chrono = "0.4"
+cloud-storage = "0.6" # 0.7+ includes request 0.11, tokio 1.4
 docopt = "1.1"
-config = "0.10"
+config = "0.11"
 env_logger = "0.8"
 futures = "0.3"
 hostname = "0.3"
 lazy_static = "1.4"
-log = { version = "0.4", features = ["max_level_debug", "release_max_level_info"] }
+log = { version = "0.4", features = ["max_level_trace", "release_max_level_info"] }
```

Comment:
I believe logging can easily get too chatty w/ this, we probably want to revert it

Reply:
Yeah, good catch. That was mostly to work out a problem I was hitting with reqwest.